src/generic/UnivPoly.jl

```diff
@@ -516,9 +516,15 @@ for op in (:+, :-, :*)
        return UnivPoly{T}($op(data(p),n), S)
     end
 
-    $op(n::Union{Integer, Rational, AbstractFloat}, p::UnivPoly) = $op(p,n)
+    function $op(n::Union{Integer, Rational, AbstractFloat}, p::UnivPoly{T}) where {T}
```

Comment:
Urgh, the UnivPoly adhoc subtraction was wrong since my recent PR #1877. Luckily the tests here caught it.